packages/dd-trace/test/plugins/versions/package.json

```diff
@@ -78,7 +78,7 @@
     "avsc": "5.7.9",
     "aws-sdk": "2.1692.0",
     "axios": "1.12.2",
-    "azure-functions-core-tools": "4.2.2",
+    "azure-functions-core-tools": "4.3.0",
```

Comment:
Should CI use this same version?

https://github.com/DataDog/dd-trace-js/blob/306888362a75a1c74b0f7fff6245e4b6022e9e0e/.github/workflows/serverless.yml#L192